Tensile/SolutionStructs.py

```diff
@@ -2068,6 +2073,27 @@ def parameterWrapper(state):
       state["EnableMatrixInstruction"] = False
 
 
+  ##############################################
+  # check and calculate Wave Seperate Global Read
+  @staticmethod
+  def checkAndAssignWaveSeperateGlobalRead(state, tc):
+    # assign WaveSeperateGlobalRead
+    if state["WaveSeperateGlobalRead%s"%tc] == -1:
+      state["WaveSeperateGlobalRead%s"%tc] = 1 if state["UnrollMajorLDS%s"%tc] else 0
+
+    # check can we use WaveSeperateGlobalRead
+    numOfWaves = state["NumThreads"] // globalParameters["WavefrontWidth"]
+    if state["WaveSeperateGlobalRead%s"%tc]:
```

Comment:
Lets not turn-on WaveSeparateGlobalRead for transposeLDS and find fractionalLoad=1 and reject solution.  user didnt ask and code turning it on and rejecting solution for fractionalLoad=1

Reply:
Hi Raman,
 Do you means to add below two conditions???
   1. set WaveSeparateGlobalRead to 0 when transposeLDS == 0
   2. reject fractionalLoad=1

Reply:
do not set WveSeparateGlobalRead=1 for Tlds=1 if WaveSeparateGLobalRead=-1; let it be controlled by user